### PR TITLE
apply_for_two_orgs.md Add new FAQ

### LINKED ISSUE #424: Add FAQ that applicants should try to apply to two orgs
Most applicants to coala will be unsuccessful, due to high demand.  For best results, applicants should also apply at a second org.
Opened by @jayvdb at [Gitter](https://gitter.im/coala/coala/gsoc?at=5a0b0bb5f257ad91099384a8)

### PR DESCRIPTION
**Pull request by amitsin6h**
Adds a new entry in the FAQ.
apply_for_two_orgs.md

Closes #424

### CODE REVIEW

**Comment by amitsin6h** [comment]
@yashovardhanagrawal please review it and merge it if their nothing any issues.

**Comment by yashovardhanagrawal** [comment]
read the [Commit Guidelines](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html)

File: _faq/apply_for_two_orgs.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+question: "Applicants should try to apply for two orgs!"
+---
+Most applicants to coala will be unsuccessful, due to high demand. So, you need to be very well prepared, as there will be 60+ applications for 10 positions, and 30 of those applications will be of very high quality. 
+
+For best results, applicants should also apply at a second organization.
```
Comment (yashovardhanagrawal) [inline _faq/apply_for_two_orgs.md]: edit this to: `As a precaution, you can apply in other organisations as well.`

and the question must be in "question" form
like: `Should I apply in two organisations?`

**Comment by yashovardhanagrawal** [comment]
you have to mention the issue in the commit.

**Comment by yashovardhanagrawal** [comment]
better commit shortlog can be:
`apply_for_two_orgs.md: Add new FAQ for to GSoC`

and description:
>This adds a new FAQ for GSoC aspirants answering
>whether they should try in two organisations or not
>
>Closes `issue_link`

**Comment by yashovardhanagrawal** [comment]
unack https://github.com/coala/projects/pull/427/commits/bc4f5f5c772cad0ae336c58fbf3345a81e8b2d96

**Comment by amitsin6h** [comment]
@yashovardhanagrawal please review it and thanks for your guidance.

**Comment by yashovardhanagrawal** [comment]
squash your commits

and still, I would say read the commit guidelines

each line in the description should be of at most 72 characters

**Comment by yashovardhanagrawal** [comment]
unack https://github.com/coala/projects/pull/427/commits/0b769e7eca6def1ed235a4addec0e3a8631bf3bf

**Comment by amitsin6h** [comment]
@yashovardhanagrawal review it.

**Comment by yashovardhanagrawal** [comment]
still your commit message is wrong I had specified the specific message that will fit here

also I asked you to squash your commits

see [git tutorial](http://api.coala.io/en/latest/Developers/Git_Basics.html) first

download GitHub Desktop if writing description feels like a problem to you

**Comment by yashovardhanagrawal** [comment]
unack https://github.com/coala/projects/pull/427/commits/a1ad854e63390485bf766ecc681a86e276213122

**Comment by amitsin6h** [comment]
@yashovardhanagrawal  but the commiy mesage and description you provided. contains 72+ characters.

**Comment by yashovardhanagrawal** [comment]
72 characters per line are allowed 😉 
you can use line break

File: _faq/apply_for_two_orgs.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+question: "Should I apply in two organisations for GSoC ?"
+---
+Most GSoC applicants to coala will be unsuccessful, due to high demand. So, you need to be very well prepared, as there will be 60+ applications for 10 positions, and 30 of those applications will be of very high quality.
```
Comment (yashovardhanagrawal) [inline _faq/apply_for_two_orgs.md]: here, edit the first line to:
> Due to high demand, GSoC selection in coala will be difficult this year. You need to be very well prepared, as there will be 60+ applications for 10 positions, and 30 of those applications will be of very high quality.

Comment (yashovardhanagrawal) [reply _faq/apply_for_two_orgs.md]: sorry I didn't see that earlier.. the answer should not be too demotivating

**Comment by amitsin6h** [comment]
@yashovardhanagrawal  please review it. Hope it gets merged now

**Comment by yashovardhanagrawal** [comment]
@amitsin6h you have written everything in a line

use line break (enter) similar to what I had used here

>This adds a new FAQ for GSoC aspirants answering
>whether they should try in two organisations or not
>
>Closes `issue_link`

**Comment by yashovardhanagrawal** [comment]
use `git commit`

vim editor will open up

edit it there using line breaks

File: _faq/apply_for_two_orgs.md (hunk 2 above)
Comment (yashovardhanagrawal) [reply _faq/apply_for_two_orgs.md]: Also you didn't apply the suggestions here

**Comment by amitsin6h** [comment]
@yashovardhanagrawal hope this does.

**Comment by yashovardhanagrawal** [comment]
the first line has 74 characters including one white space you have added

also between the description and closes `issue link` use a double line break

why don't you copy this exactly?

>This adds a new FAQ for GSoC aspirants answering
>whether they should try in two organisations or not
>
>Closes `issue_link`

**Comment by amitsin6h** [comment]
@prashant0598 @yashovardhanagrawal guys please review it know.

**Pull request closed without merging**